test/e2e/volumes.go

```diff
@@ -217,28 +218,23 @@ func testVolumeClient(client *client.Client, config VolumeTestConfig, volume api
 }
 
 var _ = Describe("Volumes", func() {
-	clean := true // If 'false', the test won't clear its namespace (and pods and services) upon completion. Useful for debugging.
+	clean := true /* if 'false', the test won't delete its pods
```

Comment:
I kind of think that we should reuse 'delete-namespace' for this. It's exactly the same use case, even though the name is different. Maybe we should rename delete-namespace into 'clean-after-tests', and use it here?